compose/.env

```diff
@@ -0,0 +1,4 @@
+MAGENTO_VERSION=
+PHP_VERSION=
+NGINX_VERSION=
+
```

Comment:
I don't see this file used or documented anyplace. What is this for, and how is it used?

Reply:
ok it's accidentally added the file